### ISSUE TITLE
Notifications and logging settings inconsistent

### ISSUE THREAD

**Issue by Gitoffthelawn** [einaregilsson/Redirector#143]
Notifications are always disabled after relaunching Firefox, whereas Logging stays enabled.   Probably best to make them consistent, and keep them both enabled.

**Comment by gkrishnaks** [comment]
Yes I had provided PR that feature request for notifications. It was an intentional choice to set Enable Notifications when browser launches as I thought it can get very annoying if notifications are always shown. For checking a specific case, notifications can be enabled and by resetting, user need not remember to toggle notifications off every time.

**Comment by Gitoffthelawn** [comment]
I see your logic, although I think the current behavior is unexpected.  I'm surprised Einar approved it, given in https://github.com/einaregilsson/Redirector/issues/80, he makes it clear that he doesn't want to revert, on startup, any action the user has taken.

My vote is to keep things consistent, and thus not revert any action the user has taken (or revert everything to defaults).

**Comment by gkrishnaks** [comment]
I did mention this approach before I sent PR here in this comment as point 2 which you 'liked' that time.. :
 https://github.com/einaregilsson/Redirector/issues/72#issuecomment-390292059

This is a different case to reduce annoyance vs major case of automatic redirect rules activation upon browser launch. 


On 24 April 2019 15:44:08 GMT+05:30, Gitoffthelawn <notifications@github.com> wrote:
>I see your logic, although I think the current behavior is unexpected. 
>I'm surprised Einar approved it, given in
>https://github.com/einaregilsson/Redirector/issues/80, he makes it
>clear that he doesn't want to revert, on startup, any action the user
>has taken.
>
>My vote is to keep things consistent, and thus not revert any action
>the user has taken (or revert everything to defaults).
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/einaregilsson/Redirector/issues/143#issuecomment-486162336

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by atomGit** [comment]
> I see your logic, although I think the current behavior is unexpected.

i agree - i want notifications to work across sessions without having to remember to enable them
